tmt/steps/__init__.py

```diff
@@ -548,8 +548,9 @@ def save(self) -> None:
 
     def wake(self) -> None:
         """ Wake up the step (process workdir and command line) """
-        # Cleanup possible old workdir if called with --force
-        if self.is_forced_run:
+        # Cleanup possible old workdir if called with --force, but not
+        # if running the step --again which should reuse saved step data
+        if self.is_forced_run and not self.opt("again"):
```

Comment:
I'd recommend adding a property similar to `is_forced_run` or `is_dry_run`. The advantage is, that it can be properly annotated while `opt()` returns `Any`, harming the linter's chances to spot a mistake. That's why I'm trying to get rid of it, converting `self.opt(...)` calls to better annotated properties :)

Reply:
Understood, makes sense. Fixed in 1ab6c961.